### PR TITLE
Add TaxBit

### PR DESCRIPTION
**Pull request by celkins**
[TaxBit](https://taxbit.com/) provides cryptocurrency tax reporting in the U.S., [supports TOTP](https://help.taxbit.com/hc/en-us/articles/4410674012311-Enabling-Two-Factor-Authentication-2FA-), and is ranked [39,689](https://www.alexa.com/siteinfo/taxbit.com).

### CODE REVIEW

File: entries/t/taxbit.com.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "TaxBit": {
+    "domain": "taxbit.com",
+    "img": "taxbit.com.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.taxbit.com/hc/en-us/articles/4410674012311-Enabling-Two-Factor-Authentication-2FA-",
```
Comment (Carlgo11) [inline entries/t/taxbit.com.json]:
```suggestion
    "documentation": "https://help.taxbit.com/hc/en-us/articles/4410674012311",
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
We prefer images as SVG files if available. Here's one that can be used:
```SVG
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 380 380"><defs><linearGradient id="a" x1="190.52" x2="190.52" y1="0" y2="381.04" gradientTransform="translate(-263.89 68.844) scale(.99363)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#135fe2"/><stop offset="1" stop-color="#24cc9a"/></linearGradient></defs><circle cx="-74.573" cy="258.14" r="190" fill="url(#a)" transform="rotate(-61.113)"/><path fill="#fff" d="M316.41 110.59h-107.5l-18.2 42.32-63 148.2h47l63-148.2h60.7zm-197.61 0h71.6l-17.9 42.32-63.6 148.2H97.799l63.601-148.2h-60.8z"/></svg>
```

File: entries/t/taxbit.com.json (hunk 1 above)
Comment (celkins) [reply entries/t/taxbit.com.json]: Thanks! I looked around their site but couldn't find an SVG version, and my naive conversion attempts looked pretty bad. Replaced the image in 5fd812d4.

**Comment by Carlgo11** [comment]
No problem @celkins. I found it at https://taxbit.com/assets/redesign/enterprise-tax/TaxBit_Tax_Center_Suite_Graphic.svg and just removed the unnecessary parts.

Just pushed another version to your branch as I saw they had done some weird math to rotate the logo. With the junk removed it's even smaller than before.

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**